examples/pom.xml

```diff
@@ -325,38 +297,6 @@
           <skip>true</skip>
         </configuration>
       </plugin>
-      <plugin>
```

Comment:
Yes, I changed the root pom to make it easy to disable test jar creation (which is not needed for this module, nor the assembly one).